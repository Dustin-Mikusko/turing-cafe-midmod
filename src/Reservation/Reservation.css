.resy-card {
  width: 200px;
  height: 250px;
  background-color: white;
  border: 5px solid teal;
  margin: 15px;
  border-radius: 10px;
}

.cancel-btn {
  height: 40px;
  width: 80%;
  background-color: teal;
  color: white;
  margin-top: 10px;
  font-size: 1.2em;
  border-radius: 1px;
}

.cancel-btn:hover {
  background-color: rgb(3, 104, 104);
  cursor: pointer;
}

.cancel-btn:active {
  transform: scale(.97);
}
