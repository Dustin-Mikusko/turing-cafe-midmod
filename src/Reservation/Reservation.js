import React from 'react';
import './Reservation.css'

const Reservation = ({ name, date, time, number}) => {
  return  (
    <article className="resy-card">
      <h2>{name}</h2>
      <p>{date}</p>
      <p>{time}</p>
      <p>Number of guests: {number}</p>
      <button type="button" className="cancel-btn">Cancel</button>
    </article>
  )
}


export default Reservation;
