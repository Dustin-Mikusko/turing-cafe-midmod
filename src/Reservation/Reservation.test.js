import React from 'react';
import { shallow } from 'enzyme';
import Reservation from './Reservation';

describe('Reservation',() => {
  it('should redner correct info matching the snapshot', () => {
    const wrapper = shallow(<Reservation />);

    expect(wrapper).toMatchSnapshot();
  })
})
