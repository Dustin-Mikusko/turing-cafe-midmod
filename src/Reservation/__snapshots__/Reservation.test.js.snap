// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reservation should redner correct info matching the snapshot 1`] = `
<article
  className="resy-card"
>
  <h2 />
  <p />
  <p />
  <p>
    Number of guests: 
  </p>
  <button
    className="cancel-btn"
    type="button"
  >
    Cancel
  </button>
</article>
`;
